website/events/forms.py

```diff
@@ -195,14 +229,31 @@ def __init__(self, *args, **kwargs):
         self.helper = FormHelper(self)
         self.helper.form_class = 'form-horizontal'
         self.helper.form_tag = False
+        self.helper.label_class = "col-sm-2"
+        self.helper.field_class = "col-sm-10"
+        self.fields['bank_entity'].required = False
+        self.fields['account_type'].required = False
+        self.fields['account_number'].required = False
+        self.fields['cbu'].required = False
 
     class Meta:
         model = Provider
         fields = [
-            'organization_name', 'document_number', 'bank_entity',
-            'account_type', 'account_number', 'cbu'
+            'organization_name',
+            'document_number',
+            'bank_entity',
+            'account_type',
+            'account_number',
+            'cbu',
         ]
 
+    def clean(self):
+        super(ProviderForm, self).clean()
+        cbu_number = self.cleaned_data.get('cbu')
+        if len(cbu_number) > 0:
```

Comment:
No estoy seguro si el valor puede ser `None`, y en ese caso tiraria una excepcion. Porque no usar:
```
if cbu_number
```

Reply:
Si, puede ser `None` porque podés querer cargar un Proveedor sin tener los datos de la cuenta bancaria.

Reply:
Pero en ese caso cbu_number is None o un empty string. Porque si es `None` el check del length va a tirar una excepcion

Reply:
🤔  pero eso no está pasando. 
Será que el `clean()` está convirtiendo el `None` en un empty string o será que el mismo form no manda un `None` y envía el empty string...

Reply:
![Screen Shot 2021-06-05 at 12 46 00](https://user-images.githubusercontent.com/6667873/120897343-01f20200-c5fc-11eb-9e43-be2291d9a6e5.png)

Bien, ahí entiendo que entonces no puede ser `None` ese valor. En todo caso termina siendo un empty string.

```python
>>> from events.form import ProviderForm
>>> data = {'organization_name': 'foo', 'document_number': '30 71480502 5', 'cbu': None}
>>> f = ProviderForm(data)
>>> f.is_valid()
True
>>> f.cleaned_data
{'organization_name': 'foo',
 'document_number': '30 71480502 5',
 'bank_entity': '',
 'account_type': '',
 'account_number': '',
 'cbu': ''}
```